src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionScopeInternal.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class ClientConnectionScopeProperties
+    {
+        public IServiceConnection ServiceConnection { get; set; }
+
+        // todo: add additional properties to flow with the scope here
+    }
+
+    internal class ServiceConnectionScopeHolder
```

Comment:
it can be a private inner class for ServiceConnectionScopeInternal ?

Reply:
Agree